app/employee-app/backend/modules/database/db_functions.bal

```diff
@@ -0,0 +1,443 @@
+    returns Employee[]|error {
+
+    stream<DBEmployee, error?> resultStream = databaseClient->query(getEmployeesQuery(filters, 'limit, offset));
+    return from DBEmployee dbEmployeeInfo in resultStream
+        select {
+            employeeId: dbEmployeeInfo.employeeId,
+            workEmail: dbEmployeeInfo.workEmail,
+            firstName: dbEmployeeInfo.firstName,
+            lastName: dbEmployeeInfo.lastName,
+            employeeThumbnail: dbEmployeeInfo.employeeThumbnail,
+            location: dbEmployeeInfo.location,
+            startDate: dbEmployeeInfo.startDate,
+            leadEmail: dbEmployeeInfo.leadEmail,
+            finalDayOfEmployment: dbEmployeeInfo.finalDayOfEmployment,
+            employeeStatus: dbEmployeeInfo.employeeStatus,
+            designation: dbEmployeeInfo.designation,
+            employmentType: dbEmployeeInfo.employmentType,
+            team: dbEmployeeInfo.team,
+            businessUnit: dbEmployeeInfo.businessUnit,
+            unit: dbEmployeeInfo.unit,
+            jobBand: dbEmployeeInfo.jobBand,
+            lead: dbEmployeeInfo.lead == 1
+        };
+}
+
+# Retrieve Organizational details from HRIS.
+#
+# + filter - Criteria to filter the data  
+# + limit - Number of records to retrieve
+# + offset - Number of records to offset
+# + return - List of business units
+public isolated function getOrgDetails(OrgDetailsFilter filter, int 'limit, int offset)
+    returns BusinessUnit[]|error {
+
+    BusinessUnit[] businessUnits = [];
+    stream<DBBusinessUnit, sql:Error?> resultStream = databaseClient->query(getOrgDetailsQuery(
+        filter = filter, 'limit = 'limit, offset = offset));
+    error? iterateError = from DBBusinessUnit bu in resultStream
+        do {
+            Department[]|error departments = bu.departments.fromJsonStringWithType();
+            if departments is error {
+                string errorMsg = string `An error occurred when retrieving departments data of ${
+                    bu.businessUnit} !`;
+                log:printError(errorMsg, departments);
+                return error(errorMsg);
+            }
+            businessUnits.push({
+                id: bu.id,
+                businessUnit: bu.businessUnit,
+                departments: departments
+            });
+
+        };
+    if iterateError is sql:Error {
+        string errorMsg = string `An error occurred when retrieving organization details!`;
+        log:printError(errorMsg, iterateError);
+        return error(errorMsg);
+    }
+    return businessUnits;
+}
+
+# Retrieve data of a specific recruit.
+#
+# + recruitId - Id of the recruitId to retrieve
+# + return - Filtered recruitId or error
+public isolated function getRecruit(int recruitId) returns types:Recruit|error {
+
+    Recruit|error result = databaseClient->queryRow(getRecruitQuery(recruitId));
+    if result is error {
+        if (result is sql:Error && result is sql:NoRowsError) {
```

Comment:
No need to have parenthesis around the if condition. Lets fix all the similar places
```suggestion
        if result is sql:Error && result is sql:NoRowsError {
```

Reply:
ACK and resolved.